plutus-metatheory/src/Algorithmic/CEK.lagda.md

```diff
@@ -321,9 +338,19 @@ V-I b {tm = suc tm} {σA = sucΠ σA} bt = V-IΠ b bt
 data Error : ∅ ⊢Nf⋆ * → Set where
   -- an actual error term
   E-error : (A : ∅ ⊢Nf⋆ *) → Error A
+```
 
+```
+data TListZipper (Γ : Ctx ∅) : (tot : List (∅ ⊢Nf⋆ *)) → ∀{vs}{h}{ts : List (∅ ⊢Nf⋆ *)} → tot ≣ vs <>> (h ∷ ts) → Set  where 
```

Comment:
here again,  I wonder if this can be described as particular kind of frame or inlined into the `Frame` type

Reply:
Again yes. Thanks for the suggestion!